deployments/optimism-goerli/usdc/configuration.json

```diff
@@ -32,6 +40,15 @@
       "liquidateCF": 0.825,
       "liquidationFactor": 0.95,
       "supplyCap": "0e18"
+    },
+    "WBTC": {
+      "address": "0xe0a592353e81a94Db6E3226fD4A99F881751776a",
+      "priceFeed": "0xC16679B963CeB52089aD2d95312A5b85E318e9d2",
+      "decimals": "8",
+      "borrowCF": 0.70,
+      "liquidateCF": 0.75,
+      "liquidationFactor": 0.95,
+      "supplyCap": "0e18"
```

Comment:
nit: this supply cap should probably be `"0e8"`, since WBTC has 8 decimals

Reply:
Ah good catch, will update it.